packages/react-core/src/helpers/resizeObserver.tsx

```diff
@@ -56,11 +83,41 @@ import { canUseDOM } from './util';
  *   );
  * }
  *
+ * Example 2b - With debounced method passed in:
+ *
+ * private inputRef = React.createRef<HTMLInputElement>();
+ * private observer: any = () => {};
+ *
+ * public componentDidMount() {
+ *   this.observer = getResizeObserver(this.inputRef.current, debounce(this.handleResize, 250), false);
+ * }
```

Comment:
Thinking we can simplify the examples? The only change is in `componentDidMount`, so perhaps we can just show how that method? For example:

```
Example 3 - With debounced method passed in

public componentDidMount() {
  this.observer = getResizeObserver(this.inputRef.current, debounce(this.handleResize, 250), false);
}
```